SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -743,7 +751,15 @@ int64 USpatialActorChannel::ReplicateActor()
 
 	bForceCompareProperties = false;		// Only do this once per frame when set
 
-	return (bWroteSomethingImportant) ? 1 : 0;	// TODO: return number of bits written (UNR-664)
+	if (ReplicationBytesWritten > 0)
+	{
+		if (USpatialNetConnection* SpatialConnection = Cast<USpatialNetConnection>(Connection))
+		{
+			SpatialConnection->AddQueuedBytes(ReplicationBytesWritten);
+		}
+	}
+
```

Comment:
driveby - can you add
`INC_DWORD_STAT_BY(STAT_NumReplicatedActorBytes, ReplicationBytesWritten);`